website/docs/general/how-to-contribute.md

```diff
@@ -0,0 +1,81 @@
+---
+id: how-to-contribute
+title: How to Contribute
+keywords:
+  - API gateway
+  - APISIX
+  - Apache APISIX
+  - contributor guide
+description: Guidelines for Apache APISIX contributors.
+---
+
+**• Engage with the Community • Build Connections • Earn Rewards**
```

Comment:
```suggestion
**• Engage with the Community • Build Connections • Earn Rewards**
```

Should we ignore the lint error?

Reply:
I think we can ignore it, it's not a formatting error.